src/Host.h

```diff
@@ -863,6 +863,8 @@ private slots:
     // return to it when switching between profiles:
     QStack<QPointer<TCommandLine>> mpLastCommandLineUsed;
 
+    bool mPendingSetFocusOnCommandLine = false;
```

Comment:
I find this a bit too wordy. How about a more concise name (`mFocusTimer`) but one with an explanation of what it does?

See this example:
![Screenshot from 2023-04-01 13-15-59](https://user-images.githubusercontent.com/110988/229285740-0ced5fa4-4b7e-4bfb-a62c-7b7d00228343.png)